src/sqlancer/postgres/gen/PostgresExpressionGenerator.java

```diff
@@ -422,6 +422,53 @@ private PostgresExpression generateTextExpression(int depth) {
             throw new AssertionError();
         }
     }
+    
+    private PostgresExpression generateWindowFunction(int depth, PostgresDataType returnType) {
+    // Generate window function arguments
+    List<PostgresExpression> arguments = new ArrayList<>();
+    if (Randomly.getBoolean()) {
+        arguments.add(generateExpression(depth + 1));
+    }
+
+    // Generate PARTITION BY expressions
```

Comment:
Instead of the comments, could we perhaps extract the relevant parts into methods with a descriptive name (e.g., here, we could have something like `generatePartitionByExpressions`?